src/scss/pages/_learn-paths.scss

```diff
@@ -0,0 +1,34 @@
+@media (min-width: 865px) {
+  .learn__path-hero {
+    height: auto;
+    max-width: initial;
+    width: 420px;
+  }
+}
+
+.learn-header {
+  display: block;
+  text-align: center;
+
+  @media (min-width: 865px) {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    text-align: initial;
+  }
+}
+
+.headline-link {
```

Comment:
1. We already have "headline" element in our design-system, so please don't reuse that name in other context.